odoo/service/server.py

```diff
@@ -869,9 +869,20 @@ def _runloop(self):
 
 class WorkerHTTP(Worker):
     """ HTTP Request workers """
+    def __init__(self, multi):
+        super(WorkerHTTP, self).__init__(multi)
+
+        # The ODOO_HTTP_SOCKET_TIMEOUT environment variable allows to control socket timeout for
+        # extreme latency situations. It's generally better to use a good buffering reverse proxy
+        # to quickly free workers rather than increasing this timeout to accomodate high network
+        # latencies & b/w saturation. This timeout is also essential to protect against accidental
+        # DoS due to idle HTTP connections.
```

Comment:
A little typo:

```suggestion
        # DDoS due to idle HTTP connections.
```

Reply:
It really means a basic Denial of Service, not a Distributed one, since you can deadlock your whole server by yourself just by printing a report, if you're too low on workers.

Reply:
Oh OK, I thought it was just a typo.